@Types/ITechRecord.d.ts

```diff
@@ -155,35 +207,50 @@ interface Axle {
   };
 }
 
+interface Dda {
+  certificateIssued: boolean;
+  wheelchairCapacity: number;
+  wheelchairFittings: string;
+  wheelchairLiftPresent: boolean;
+  wheelchairLiftInformation: string;
+  wheelchairRampPresent: boolean;
+  wheelchairRampInformation: string;
+  minEmergencyExits: number;
+  outswing: string;
+  ddaSchedules: string;
+  seatbeltsFitted: string;
+  ddaNotes: string;
+}
+
```

Comment:
Why do I see the same exact changes in the branch where you are about to merge this  (feature/CVSB-10813) ?

Reply:
Some of the tickets are branched off 10813, because I needed those changes to be able to implement the rest of the tickets, and this PR is created against 10813 so we can only see the changes done in this ticket. When 10813 will be merged in develop I'll change the base.

Reply:
Oh, looks like cvs-ops force pushed some changes...I think I have to do a rebase

Reply:
should be ok now